build.gradle.kts

```diff
@@ -103,6 +111,62 @@ val requireNativeTask by tasks.register("requireNativeDirectory") {
 	}
 }
 
-tasks.withType<KotlinCompile>().all {
+val jExtractDownloadFile = layout.buildDirectory.file("resources/jextract.tar.gz")
```

Comment:
I assume there's a better way to link outputs from one task as inputs to another.